src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -291,7 +298,13 @@ public void periodic() {
         RigidTransform2 pose = getPose();
         odometryXEntry.setDouble(pose.translation.x);
         odometryYEntry.setDouble(pose.translation.y);
-        odometryAngleEntry.setDouble(getPose().rotation.toDegrees());
+        odometryAngleEntry.setDouble(pose.rotation.toDegrees());
+        if (follower.getLastState() != null) {
+            //0.0254 is inches to meter conversion
+            field.setRobotPose(follower.getLastState().getPathState().getPosition().x *0.0254, follower.getLastState().getPathState().getPosition().y * 0.0254, Rotation2d.fromDegrees(follower.getLastState().getPathState().getRotation().toDegrees()));
```

Comment:
Consider making 0.0254 a constant to make it clear what it is converting to and from.